### PR TITLE
New spec "ipcs -s -i xxx"

### PR DESCRIPTION
**Pull request by huali027**
Signed-off-by: Huanhuan Li <huali@redhat.com>

### All Pull Requests:

Check all that apply:

* [ x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:
Add the spec back since it is used by an existing webserver rule.

### CODE REVIEW

**Review by xiangce — COMMENTED** [review]
Better remove the verb `get_` from the filename and the name of the spec, and use `ipcs` as the filename, `semid` as the name of spec.

File: insights/specs/datasources/get_semid.py
Comment (xiangce) [inline insights/specs/datasources/get_semid.py:1]: Please add a title for this module

File: insights/specs/datasources/get_semid.py
Comment (xiangce) [inline insights/specs/datasources/get_semid.py:3]: If there is possible to get the expected list from an existing spec, do not use the Parser which is heavier.

The existing spec `ipcs_s` is quite simple, and can be used directly for this datasource.

**Review by xiangce — CHANGES_REQUESTED** [review]

File: insights/collect.py
Comment (xiangce) [inline insights/collect.py:183]: By change to base on the `ipcs_s` spec, this pre-check parser is not required.

File: insights/specs/datasources/ipcs.py
Comment (xiangce) [inline insights/specs/datasources/ipcs.py:44]: How about skipping it when nothing is get from the `ipcs -s`?

**Review by xiangce — APPROVED** [review]
Looks good. But it would be better if the datasource `semid` could handle the empty result itself.

Approved pending the approval of spec.

**Comment by huali027** [comment]
> Looks good. But it would be better if the datasource `semid` could handle the empty result itself.
> 
> Approved pending the approval of spec.

Sure, I'll check if it's emtpy, thanks for the review.

**Review by xiangce — APPROVED** [review]
Approved pending the approval of Spec

File: insights/specs/default.py
Comment (bfahr) [inline insights/specs/default.py:451]: @huali027 this spec can be left in the original form as a `simple_command`.  There's no need to create the LocalSpec in the datasource.

File: insights/specs/datasources/ipcs.py
Comment (bfahr) [inline insights/specs/datasources/ipcs.py:15]: There's no need to create this, just reference the spec `insights.specs.Specs.ipcs_s` instead.

**Review by bfahr — CHANGES_REQUESTED** [review]

File: insights/specs/datasources/ipcs.py
Comment (huali027) [reply insights/specs/datasources/ipcs.py]: @bfahr, sure, updated, thanks.

**Review by bfahr — APPROVED** [review]

**Pull request merged**
